visualizer/index.js

```diff
@@ -24,6 +24,25 @@ module.exports = function (trees, opts) {
   })
   const { colors } = flamegraph
 
+  let userZoom = true // false if the last zoom call was initiated by 0x
+  flamegraph.on('zoom', (d) => {
+    if (!userZoom) {
+      userZoom = true
+      return
+    }
+
+    focusNode(d)
+  })
+  window.addEventListener('popstate', (event) => {
+    userZoom = false
+    jumpToState(event.state || {
+      // No hash anymore, jump to root node (0) but don't change settings
+      merged: state.control.merged,
+      excludeTypes: Array.from(state.typeFilters.exclude),
+      nodeId: 0,
```

Comment:
Tiny style nit: I personally don't have any issue with trailing commas in objects, but since there's currently no lint or style guide fixed for this repo, it's probably best to stick with the existing convention which appears to have been not adding trailing commas.